setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name='truffleHog',
+    version='1.0.0',
+    description='Searches through git repositories for high entropy strings, digging deep into commit history.',
+    url='https://github.com/dxa4481/truffleHog',
+    author='Dylan Ayrey',
+    author_email='dxa4481@rit.edu',
+    license='GNU',
+    packages =['truffleHog'],
+    install_requires=[
+        'GitPython == 2.1.1'
```

Comment:
Is exactly 2.1.1 required? 

A tool that would be installed globally should rather not enforce a particular version IMO.